src/DetailsViewPanel.ts

```diff
@@ -112,8 +114,8 @@ export default class DetailsViewPanel {
             this.nextHistory = [];
         }
 
-        baseUrl = baseUrl || "https://salesforcecommercecloud.github.io";
-        const contentUrl = normalizeUrl(`${baseUrl}/${topic}`);
+        const contentUrl = topic.startsWith(DOCS_BASE) ? normalizeUrl(topic) : normalizeUrl(`${DOCS_BASE}${topic}`);
+        const pageUrl = contentUrl.replace('?embed=true', '');
```

Comment:
This updates the link at the top of the website panel, and removes the embed param if it exists so when it is clicked, they get the full site again.